product_docs/docs/pem/9/installing/configuring_the_pem_server_on_linux.mdx

```diff
@@ -22,24 +22,25 @@ The PEM server package includes a script (`configure-pem-server.sh`) to help aut
+| `-sp`  or `--superpassword`              | Superuser password of the PEM database server. This value is required.                                                                                                                                                                                                              |
+| `-su`  or `--superuser`                  | Superuser name of the PEM database server.                                                                                                                                                                                                                                          |
+| `-au`  or `--use-agent-user`             | PEM agent user name.                                                                                                                                                                                                                                                                |
+| `-t`   or `--type`                       | Installation type: Specify `1` if the configuration is for web services and backend database, `2` if you're configuring web services, or `3` if you're configuring the backend database. If you specify `3`, the database must reside on the local host.                            |
+| `-un`  or `--uninstall-pem-server`       | Uninstalls the PEM server.                                                                                                                                                                                                                                                          |
+| `-nhc` or `--no-hba-change`              | Skips the changes done to `pg_hba.conf` and `pg_config` files.                                                                                                                                                                                                                      |
+| `-uac` or `--use-agent-sslcert`          | Reuses the existing agent SSL certificate while configuring the PEM server.                                                                                                                                                                                                         |
+| `-uak` or `--use-agent-sslkey`           | Reuses the existing agent SSL key while configuring the PEM server.                                                                                                                                                                                                                 |
+| `-scs` or `--server-certificate-subject` | Provides the custom web server certificate subject. The default value includes the EDB details. The format is `/C=CountryCode/ST=StateName/L=CityName/O=CompanySelectionName/CN=CommonNameorHostname/emailAddress=user@company.com`. Provide the `C=CountryCode` as `Alpha-2` code. |
```

Comment:
We are not providing any default value, on fresh installation of 9.5 customer has to input the server-certificate-subject else the script will fail

Reply:
removed the default value